matchzoo/preprocessor/cdssm_preprocessor.py

```diff
@@ -164,8 +167,11 @@ def transform(
             text = [ngram_unit.transform([term]) for term in text]
             text = [hash_unit.transform(term) for term in text]
             text = list(itertools.chain(*text))
+            length = len(text)
             text = fix_unit.transform(text)
+            length = min(length, self._text_length) * self._dim_ngram
```

Comment:
`text_length = min(len(text), ....` and remove 170

Reply:
Here, it cannot be formalized as you suggested since the `text` have been changed in line 171. Thus, the `length` in line 170 is not equal with `len(text)` in line 172.

Reply:
rename text in line 171 then

Reply:
I think it is more appropriate to keep the name `text` to keep consistence.